src/vita-make-fself.c

```diff
@@ -4,7 +4,11 @@
 #include <inttypes.h>
 #include <zlib.h>
 
+typedef struct {} vita_export_t;
```

Comment:
This is a hack to get it to compile. Ideally we don't have headers that depend on structures defined in other headers but refactoring all that is too much work for this.